src/esp/nav/PathFinder.cpp

```diff
@@ -1625,7 +1628,44 @@ PathFinder::Impl::getTopDownView(const float metersPerPixel,
   for (int h = 0; h < zResolution; ++h) {
     for (int w = 0; w < xResolution; ++w) {
       vec3f point = vec3f(curx, height, curz);
-      topdownMap(h, w) = isNavigable(point, 0.5);
+      topdownMap(h, w) = isNavigable(point, eps);
+      curx = curx + metersPerPixel;
+    }
+    curz = curz + metersPerPixel;
+    curx = startx;
+  }
+
+  return topdownMap;
+}
+
+typedef Eigen::Matrix<int, Eigen::Dynamic, Eigen::Dynamic> MatrixXi;
+
+MatrixXi PathFinder::Impl::getTopDownIslandView(const float metersPerPixel,
+                                                const float height,
+                                                const float eps) const {
+  std::pair<vec3f, vec3f> mapBounds = bounds();
+  vec3f bound1 = mapBounds.first;
+  vec3f bound2 = mapBounds.second;
```

Comment:
```suggestion
  vec3f bound1 = std::move(mapBounds.first);
  vec3f bound2 = std::move(mapBounds.second);
```

Not sure if they are trivially copyable (so double check with clang-tidy), but I seem to remember them not being non-trivially movable.